gnucash/register/ledger-core/split-register-control.cpp

```diff
@@ -846,59 +848,59 @@ gnc_split_register_auto_completion (SplitRegister *reg,
         const char *desc;
 
         /* there must be a blank transaction * */
-        if (blank_trans == NULL)
-            return FALSE;
+        if (blank_trans == nullptr)
+            return false;
 
         /* we must be on the blank split */
         if (trans != blank_trans)
-            return FALSE;
+            return false;
 
         /* and leaving the description cell */
         if (!gnc_cell_name_equal (cell_name, DESC_CELL))
-            return FALSE;
+            return false;
 
         /* nothing but the date, num, and description should be changed */
         /* FIXME, this should be refactored. */
         if (gnc_table_layout_get_cell_changed (reg->table->layout,
-                                               XFRM_CELL, TRUE)  ||
+                                               XFRM_CELL, true) ||
                 gnc_table_layout_get_cell_changed (reg->table->layout,
```

Comment:
All the function calls should line up with the first one.

Reply:
Changed.